ktor-utils/jvm/test/io/ktor/tests/utils/DateJvmTest.kt

```diff
@@ -0,0 +1,37 @@
+package io.ktor.tests.utils
+
+import io.ktor.util.date.*
+import java.text.SimpleDateFormat
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+import java.util.*
+import kotlin.test.*
+
+class DateJvmTest {
+    @Test
+    fun testJvmDate() {
+        val dateRaw = GMTDate(1346524199000)
+        val date = dateRaw.toJvmDate()
+
+        assertEquals(dateRaw.timestamp, date.time)
+    }
+
+    @Test
+    fun testJvmDateWithOffSetDateTime() {
+        val dateRaw = GMTDate(20, 20, 20, 20, Month.FEBRUARY, 20)
+        val date: OffsetDateTime = OffsetDateTime.of(dateRaw.year, dateRaw.month.ordinal, dateRaw.dayOfMonth, dateRaw.hours, dateRaw.minutes, dateRaw.seconds, 0, ZoneOffset.UTC)
+
+        assertEquals(dateRaw.timestamp, date.toEpochSecond())
```

Comment:
It looks like you try to compare different quantities: 
```
java.lang.AssertionError: expected:<-61531846780000> but was:<-61534352380>
```

Reply:
Yeah, my bad, sorry.

Fixed, added 1 test.
```
BUILD SUCCESSFUL in 18s
123 actionable tasks: 4 executed, 119 up-to-date
```